doc/content/gateways/COTX-X1 Lite gateway

```diff
@@ -0,0 +1,47 @@
+---
+title: "COTX X1 Lite"
+description: ""
+---
+
+COTX X1 Lite is a standard LoRaWAN indoor gateway and a Helium data-only hotspot.
+It is highly compatible and has a user-friendly interface.
+Rich in software tools, powerful back office management and technical support.
+Compact and exquisite appearance, easy to install and deploy.
+![Lite-1.png](./TTN-X1%20Lite-1.png)
+
+##Key Features 
+A complete 8-channel gateway based on the LORAWAN protocol.
+LoRa band support：CN470,EU868,US915,AS923,KR920,AU915.
+LoRa power support up to 27dbm.
+Built-in Ethernet. Supports WiFi-AP mode as well as LTE-CAT4 uplink connectivity for easy adaptation to different scenarios.
+Gateway management and configuration based on OPENWRT, supporting MQTT bridging mode for easy deployment of gateways and integration environments.
+
+##Provisioning on TTN Network Server 
+1.1 Required Equipment 
+COTX X1 Lite Gateway
+Power source
+Internet ready
+
+##1.2 Gateway Setup 
+Step 1: First  insert the SMA male connector of the antenna feeder into the LoRa®  Antenna port of the device and tighten it.
+![Lite-3.png](./TTN-X1%20Lite-3.png)
```

Comment:
same as explained above, change the file name to something simpler and add the image using the `figure` tag